docs/source/ref/models/search.rst

```diff
@@ -0,0 +1,626 @@
+
+Atlas search expressions ease the use of MongoDB Atlas search's :doc:`full text
+and vector search engine <atlas:atlas-search>`.
+
+For the examples in this document, we'll use the following models::
+
+    from django.db import models
+    from django_mongodb_backend.models import EmbeddedModel
+    from django_mongodb_backend.fields import ArrayField, EmbeddedModelField
+
+
+    class Writer(EmbeddedModel):
+        name = models.CharField(max_length=10)
+
+
+    class Article(models.Model):
+        headline = models.CharField(max_length=100)
+        number = models.IntegerField()
+        body = models.TextField()
+        location = models.JSONField(null=True)
+        plot_embedding = ArrayField(models.FloatField(), size=3, null=True)
+        writer = EmbeddedModelField(Writer, null=True)
+
+``SearchEquals``
+----------------
+
+.. class:: SearchEquals(path, value, *, score=None)
+
+Matches documents where a field is equal to a given value.
+
+Uses the :doc:`equals operator <atlas:atlas-search/equals>` to perform exact
+matches on fields indexed in a MongoDB Atlas Search index.
+
+.. code-block:: pycon
+
+    >>> from django_mongodb_backend.expressions import SearchEquals
+    >>> Article.objects.annotate(score=SearchEquals(path="headline", value="title"))
+    <QuerySet [<Article: headline: title>]>
+
+The ``path`` argument can be either the name of a field (as a string), or a
+:class:`~django.db.models.F` instance.
+
+The ``value`` argument must be a string or a :class:`~django.db.models.Value`.
+
+An optional ``score`` :class:`SearchScoreOption` argument to tune the
+relevance score.
+
+``SearchAutocomplete``
+----------------------
+
+.. class:: SearchAutocomplete(path, query, *, fuzzy=None, token_order=None, score=None)
+
+Enables autocomplete behavior on string fields.
+
+Uses the :doc:`autocomplete operator <atlas:atlas-search/autocomplete>` to
+match the input query against a field indexed with ``"type": "autocomplete"``
+in a MongoDB Atlas Search index.
+
+.. code-block:: pycon
+
+    >>> from django_mongodb_backend.expressions import SearchAutocomplete
+    >>> Article.objects.annotate(score=SearchAutocomplete(path="headline", query="harry"))
+    <QuerySet [
+       <Article: title: Harry and the History of Magic>,
+       <Article: title: Harry Potter’s Cultural Impact on Literature>
+    ]>
+
+The ``path`` argument specifies the field to search and can be a string or a
+:class:`~django.db.models.F`.
+
+The ``query`` is the user input string to autocomplete and can be passed as a
+string or a :class:`~django.db.models.Value`.
+
+Optional arguments:
+
+- ``fuzzy``: A dictionary with fuzzy matching options such as
+  ``{"maxEdits": 1}``.
+- ``token_order``: Controls token sequence behavior. Accepts values like
+  ``"sequential"`` or ``"any"``.
+- ``score``: A :class:`SearchScoreOption` to tune the relevance score.
+
+``SearchExists``
+----------------
+
+Atlas Search expression that matches documents where a field exists.
```

Comment:
Adjust the style:

chop "Atlas Search expression that ..." prefixes in favor of "Matches..." and "Uses..." (next paragraph).